charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.52.4
+version: 0.53.0
```

Comment:
the version can be 0.52.5 as there no breaking change or operator bump with this pr

Reply:
Given the operator release today, I elected to bump the operator version instead.

Reply:
Actually, scratch that, after pulling in the lastest helm chart version, I'm bumping this to 0.53.1 since that should be the next Helm chart version.